python/interpret_community/common/explanation_utils.py

```diff
@@ -17,8 +17,10 @@
 with warnings.catch_warnings():
     warnings.filterwarnings('ignore', 'Starting from version 2.2.1', UserWarning)
     import shap
-    from shap.common import DenseData
-
+    try:
+        from shap.common import DenseData
+    except ImportError:
+        from shap.utils._legacy import DenseData
```

Comment:
As discussed in other PR.. We probably shouldn't take a dependency on deprecated interfaces on purpose..

Reply:
Other PR here: https://github.com/interpretml/interpret-community/pull/370

Reply:
we already support other types of data, numpy, pandas, etc.  We also support shap's DenseData type, which is output in some cases (like when using shap.kmeans)

Reply:
Talked offline and @imatiach-msft agreed to remove Dense data support from interpret-community in a future PR